### PR TITLE
[VEN-2413]: vip to increase wbeth borrow cap to 4000

### PR DESCRIPTION
**Pull request by coreyar**
## Description

Adds VIP per Chaos Labs recommendations to increase the WBETH borrow cap to 4000

### CODE REVIEW

File: simulations/vip-260/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+forking(36334143, () => {
+  const provider = ethers.provider;
+  let comptroller: ethers.Contract;
+
+  before(async () => {
+    comptroller = new ethers.Contract(COMPTROLLER, DIAMOND_CONSOLIDATED_ABI, provider);
+  });
+
+  describe("Pre-VIP behavior", () => {
+    it("Verify current borrow cap is 200", async () => {
```
Comment (chechu) [inline simulations/vip-260/bscmainnet.ts:19]:
```suggestion
    it("Verify current borrow cap is 2000", async () => {
```

File: simulations/vip-260/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  });
+
+  testVip("VIP-260 Set new WBETH borrow cap to 4000", vip260(), {
+    callbackAfterExecution: async (txResponse: TransactionResponse) => {
+      await expectEvents(txResponse, [DIAMOND_CONSOLIDATED_ABI], ["NewBorrowCap"], [1]);
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("Verify current borrow cap is 200", async () => {
```
Comment (chechu) [inline simulations/vip-260/bscmainnet.ts:32]:
```suggestion
    it("Verify current borrow cap is 4000", async () => {
```

File: simulations/vip-260/bscmainnet.ts (hunk 1 above)
Comment (coreyar) [reply simulations/vip-260/bscmainnet.ts:19]: It actually is set to 200 on testnet

File: src/vip-framework/index.ts [hunk 3]
```diff
@@ -130,12 +130,13 @@ export const testVip = (description: string, proposal: Proposal, options: Testin
       await mineUpTo((await ethers.provider.getBlockNumber()) + VOTING_PERIOD + 1);
       const tx = await governorProxy.connect(proposer).queue(proposalId);
       await tx.wait();
+      console.log(await governorProxy.state(proposalId), "state");
```
Comment (chechu) [inline src/vip-framework/index.ts:133]: Do we want to keep this? It seems a debug trace. `state` should be always `5` here

File: vips/vip-260/bsctestnet.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  return makeProposal(
+    [
+      {
+        target: COMPTROLLER,
+        signature: "_setMarketBorrowCaps(address[],uint256[])",
+        params: [[WBETH_VTOKEN], [NEW_BORROW_CAP]],
+      },
+    ],
+    meta,
+    ProposalType.REGULAR,
```
Comment (GitGuru7) [inline vips/vip-260/bsctestnet.ts:39]:
```suggestion
    ProposalType.FAST_TRACK,
```

File: simulations/vip-260/bscmainnet.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  });
+
+  testVip("VIP-260 Set new WBETH borrow cap to 4000", vip260(), {
+    callbackAfterExecution: async (txResponse: TransactionResponse) => {
+      await expectEvents(txResponse, [DIAMOND_CONSOLIDATED_ABI], ["NewBorrowCap"], [1]);
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("Verify current borrow cap is 2000", async () => {
```
Comment (GitGuru7) [inline simulations/vip-260/bscmainnet.ts:32]:
```suggestion
    it("Verify current borrow cap is 4000", async () => {
```

File: simulations/vip-260/bsctestnet.ts [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+      const currentWBethBorrowCap = await comptroller.borrowCaps(WBETH_VTOKEN);
+      expect(currentWBethBorrowCap).equals(OLD_BORROW_CAP);
+    });
+    it("Verify current borrow cap is 200", async () => {
+      await pretendExecutingVip(vip260());
+      const currentWBethBorrowCap = await comptroller.borrowCaps(WBETH_VTOKEN);
+      expect(currentWBethBorrowCap).equals(NEW_BORROW_CAP);
+    });
+  });
+});
```
Comment (GitGuru7) [inline simulations/vip-260/bsctestnet.ts:34]: Seems redundant?

File: simulations/vip-260/bsctestnet.ts [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+  testVip("VIP-260 Set new WBETH borrow cap to 4000", vip260(), {
+    supporter: "0x55a9f5374af30e3045fb491f1da3c2e8a74d168d",
+    callbackAfterExecution: async (txResponse: TransactionResponse) => {
+      await expectEvents(txResponse, [DIAMOND_CONSOLIDATED_ABI], ["NewBorrowCap"], [1]);
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("Verify current borrow cap is 200", async () => {
```
Comment (GitGuru7) [inline simulations/vip-260/bsctestnet.ts:59]:
```suggestion
    it("Verify current borrow cap is 4000", async () => {
```

File: vips/vip-260/bsctestnet.ts (hunk 4 above)
Comment (chechu) [reply vips/vip-260/bsctestnet.ts:39]: FAST_TRACK is not allowed on testnet for this command :(

File: simulations/vip-260/bsctestnet.ts (hunk 6 above)
Comment (coreyar) [reply simulations/vip-260/bsctestnet.ts:34]: yeah it is I added it while debugging while FastTrack was failing

Comment (GitGuru7) [reply simulations/vip-260/bsctestnet.ts]: I think we can remove it.

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/261?chainId=56

**Pull request merged**